Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/ActionHandlers/ActUIElementHandler.cs

```diff
@@ -478,5 +489,57 @@ private async Task HandleSetValueOperationAsync()
             string value = _act.GetInputParamCalculatedValue("Value");
             await element.SetAttributeValueAsync(name: "value", value);
         }
+
+        private async Task HandleClickAndValidateAsync()
+        {
+            IBrowserElement elementToClick = await GetFirstMatchingElementAsync();
+            ActUIElement.eElementAction clickType = Enum.Parse<ActUIElement.eElementAction>(_act.GetInputParamValue(ActUIElement.Fields.ClickType).ToString());
+
+            if (clickType == ActUIElement.eElementAction.Click)
+            {
+                await elementToClick.ClickAsync();
+            }
+            else if (clickType == ActUIElement.eElementAction.JavaScriptClick)
+            {
+                string script = "element => element.click()";
+                await elementToClick.ExecuteJavascriptAsync(script);
+            }
+            else
+            {
+                _act.Error = $"Operation '{clickType}' is not supported by Playwright driver";
+            }
+            eLocateBy validateElementLocateBy = Enum.Parse<eLocateBy>(_act.GetInputParamValue(ActUIElement.Fields.ValidationElementLocateBy).ToString());
+            string validationElementLocatorValue = _act.GetInputParamValue(ActUIElement.Fields.ValidationElementLocatorValue).ToString();
+            IEnumerable<IBrowserElement> validationElementSearchResult = await _elementLocator.FindMatchingElements(validateElementLocateBy, validationElementLocatorValue);
+            IBrowserElement? elementToValidate = validationElementSearchResult.FirstOrDefault();
+            if (elementToValidate == null)
```

Comment:
Use .Any() or .Count to check if at least one element is present in the list instead of relying on FirstOrDefault().

Reply:
done